src/components/emailForm.js

```diff
@@ -0,0 +1,97 @@
+import React, {Component} from 'react';
+import * as actions from '../actions/index';
+import {connect} from 'react-redux';
+import { reduxForm, Field} from 'redux-form';
+import {reset} from 'redux-form';
+import TextareaAutosize from 'react-textarea-autosize';
+import Captcha from './captcha.js';
+import ReCAPTCHA from "react-google-recaptcha";
+
+
+
+class EmailForm extends Component {
+  constructor (props){
+    super (props);
+    this.state = {
+      messageSent: false
+    }
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick (event){
+    event.preventDefault();
+    const message = {
+      message: this.message.value,
+      email: this.email.value,
+      name: this.name.value
+    }
+
+    console.log('this is the message', message)
+    this.props.dispatch(
+      actions.sendMessageToMaria(message)
+    )
+
+    this.message.value = '';
+    this.email.value = '';
+    this.name.value = '';
+  }
+
+  handleSubmit(formProps){
+      this.props.signupUser(formProps);
+  };
+
+  onChange(value) {
```

Comment:
I've changed it, bit it is very likely that I won't use this for the captcha. It was just a test.